dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -268,16 +259,16 @@ public SolrQueryResponse search(DataverseRequest dataverseRequest, List<Datavers
          *
          */
 
-        //I'm not sure if just adding null here is good for hte permissions system... i think it needs something
+        
+        // Logged in user, has publication status facet
```

Comment:
Imo little private methods > comments, you can extract `dataverseRequest.getUser().isAuthenticated()` to be `hasPublicationStatus(User user)` then the method explains itself.

Reply:
Well. I will remove comment because I think code is already self explanatory. As to separate method I don't think that `hasPublicationStatus` name is good because it could make reader think that something is already applied.